Automattic/wp-calypso
# Move e2e test triggers in to WP-Calypso

#### Changes proposed in this Pull Request

* Moves the e2e test triggers from various wrapper repos in to wp-calypso where the tests now reside

#### Testing instructions
Verify that the configs look correct to you and have the right parameters needed to run the tests.
